packages/react-components/src/components/VideoGallery/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '../DevicesButton';
+
+export interface LocalVideoCameraButtonProps {
+  /** Array of cameras available to the user. */
+  cameras?: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  currentCamera?: OptionsDevice;
+  /** callback function to change video feed. */
+  cycleCamera: (device: OptionsDevice) => Promise<void>;
+}
+/**
+ * local video tile camera cycle button - for use on mobile screens only.
+ * @param props
+ * @returns
+ * @private
+ */
+export const LocalVideoCameraButton = (props: LocalVideoCameraButtonProps): JSX.Element => {
+  const { cameras, currentCamera, cycleCamera } = props;
+  const localCameraSwitcherStyles: IButtonStyles = {
+    root: {
+      position: 'absolute',
+      right: '0',
+      color: 'white'
```

Comment:
Is it true that this button is _only_ ever shown when the local video feed is on? In that case, it makes sense to not respect the `theme`.

Perhaps add a comment here: "Always shown on top of local video feed, so force a specific color instead of using the `theme`".

Reply:
yes it only ever shown when the local video feed is on as well it is intended but mobile use primarily as well.